### PR TITLE
Primera prueba

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
Hello,

Thanks for your time and support, here goes my part on the first part - experimenting with Axios HTTP calls. the HEAD requirement was pretty exciting, from my understanding since it is used to check responses without the need of retrieving anything e.g files, etc. So I used it for a basic authentication, quite cool. Also, I used params to create data on the server with a specific id, thinking on passing this id on the DELETE call.

Please excuse me if there are still problems with EOF, I checked before pushing but I am not sure.

Cheers,

Sebastian

### CODE REVIEW

**Review by manuelq12 — APPROVED** [review]
Well you certainly did a great job @SebastianGomezVidal  💯 
Thanks for the detailed description and statements of your work, they were really helpful 👍

File: test/MyFirstApiConsume.test.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+      //Using chai for assertions
+      expect(response.status).to.equal(StatusCodes.OK);
+      expect(response.statusText).to.equal('OK');
+      expect(response.data).to.have.property('url').to.equal('https://httpbin.org/delete');
+      });
+  });
+
+});
+
```
Comment (manuelq12) [inline test/MyFirstApiConsume.test.js:164]: Can you erase this blank line before merging?

File: test/MyFirstApiConsume.test.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      //POST call using axios plus parameters
+      const response = await axios.delete('https://httpbin.org/delete',null, {params: { id: "2" }});
+
+      //Using chai for assertions
+      expect(response.status).to.equal(StatusCodes.OK);
+      expect(response.statusText).to.equal('OK');
+      expect(response.data).to.have.property('url').to.equal('https://httpbin.org/delete');
+      });
+  });
+
```
Comment (manuelq12) [inline test/MyFirstApiConsume.test.js:162]: Can you erase this blank line before merging?

**Comment by manuelq12** [comment]
Remember to keep english everywhere, in this case, the title

File: test/MyFirstApiConsume.test.js (hunk 1 above)
Comment (SebastianGomezVidal) [reply test/MyFirstApiConsume.test.js]: done

File: test/MyFirstApiConsume.test.js (hunk 2 above)
Comment (SebastianGomezVidal) [reply test/MyFirstApiConsume.test.js]: done

**Pull request merged**